static-assets/components/cstudio-admin/mods/content-types.js

```diff
@@ -2696,7 +2696,7 @@
 
           var updatePropertyFn = function (name, value) {
             for (var l = item.properties.length - 1; l >= 0; l--) {
-              if (item.properties[l].name === name) {
+              if (item.properties[l].name === name && item.properties[l].value !== value) {
```

Comment:
This seems to fix the image picker only. There seems to be a problem in the "changes pending dialog": why pressing accept doesn't close?